apps/web/src/components/Publication/DecryptedPublicationBody.tsx

```diff
@@ -189,6 +190,13 @@ const DecryptedPublicationBody: FC<DecryptedPublicationBodyProps> = ({
     setIsDecrypting(false);
   };
 
+  if (isIOS && isMobile && showMore) {
```

Comment:
isIOS will be mobile at the end right?

Reply:
I'd suggest to use some other solution instead of hard detecting things, maybe line-clamp in tailwind?

Reply:
1. isIOS will be true for Mac, iPhone and iPad.
2. Tailwind line-clamp seems to work with iphone browsers, but only in case, when wrapper element has a single div.(As mentioned in PR details above) . How can this very specific issue in iphone browser be handled without detecting the user's device?
PS: This issue occurs in all browsers in iPhone & iPad.  (That's why isIOS & isMobile)
https://github.com/duskload/react-device-detect/blob/master/docs/selectors.md